code.gs

```diff
@@ -90,6 +91,7 @@ var settings = {
      * By default the script only tracks birthday events.
      */
     eventTypes: {
+      Birthdays: true,
```

Comment:
Could you elaborate on why is this needed?

Reply:
this is the way the event is handled in the API, at least for me it was when i needed to add 'additional' birthdays date, e.g. a kids birthday of my contact. the all caps handle didnt work for me but i didnt change it due to backward compatability

Reply:
line 55 was extended to elaborate/explain that the script is not triggered at e.g. 6:00 but sometime between 6 and 7

Reply:
This is the expected behaviour: the BIRTHDAY event is the birthday of the contact, and ANNIVERSARY is the anniversary. These exist because they are specified by Google. Any othe event falls in the CUSTOM event. By setting CUSTOM: true you should be seeing all the additional events you set up.

Let me know if this is the case.

Reply:
custom didnt work for me, thats why added this new one

Reply:
That's strange...
Still we cannot add custom names in the base source code which everyone uses
Please remove this line so I can merge this PR: if you want you can then open an issue where we can investigate your problem further, since it looks like a bug.